### DISCUSSION TITLE
Active / deactivate automations

### DISCUSSION THREAD

**Discussion by TurboKanne** [andesse/hue-clip-api.node-red-flows#11] [General]
Hi,

I installed this peace and for the lights and groups it works great.
is there a way to use this node red portion for activating / deactivating automations in the official hue app (hue bridge)?

that would be very helpful.

best regards
TurboKanne

**Comment by andesse** [comment]
Das sind die resourcen aus der Api die automationen steuern. Da musst Du mal nachschauen was alles zur Verfügung steht und ob deine Automation mit dabei ist. Such mal nach dem Namen. Damit hab ich mich nicht beschäftigt. 

![IMG_2493](https://user-images.githubusercontent.com/76150626/182365872-e435424c-8b2e-42e3-bfd3-0774ed8ffc95.jpeg)
